### PR TITLE
Customer Home: Make Site Tools Card Conditional

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

This makes the change suggested by @kwight in #39075: 

> I'm wondering about the component being conditional on is_mobile() (eg. if mobile, display FoldableCard, otherwise, display Card)? It would be more verbose, but magical whitespace values are a little harder to reason about when reading the code.

#### Testing instructions

This should be the same as #39075, as there should really be no visible changes. It may also be worth confirming that the still buttons work.

### CODE REVIEW

File: client/my-sites/customer-home/main.jsx [hunk 1]
```diff
@@ -325,24 +325,126 @@ class Home extends Component {
 		);
 	}
 
+	renderSiteTools() {
```
Comment (sixhours) [inline client/my-sites/customer-home/main.jsx]: Nice idea to move this into a separate function for easier reading. 👍

File: client/my-sites/customer-home/main.jsx [hunk 2]
```diff
@@ -361,105 +463,26 @@ class Home extends Component {
 					{ // "Go Mobile" has the highest priority placement when viewed in smaller viewports, so folks
 					// can see it on their phone without needing to scroll.
 					isMobile() && <GoMobileCard /> }
-					{ displayChecklist ? (
+					{ displayChecklist && (
```
Comment (sixhours) [inline client/my-sites/customer-home/main.jsx]: We only want to show Site Tools after the checklist has been completed; with this change, the Site Tools display as an empty card underneath the checklist:

Screen Shot 2020-01-28 at 4 39 48 PM

The logic should look like:

```
if ( displayChecklist ) { 
    //show the checklist
} else {
    //show site tools
}
```

File: client/my-sites/customer-home/main.jsx (hunk 1 above)
Comment (sixhours) [inline client/my-sites/customer-home/main.jsx]: Good idea to move this to its own function. Would it make sense to move the `Card`/`FoldableCard` logic here as well? Then the only output in the final render function would be `{ this.renderSiteTools() }` and everything is kept in one place. Not a blocker, IMO.

**Comment by Aurorum** [comment]
Both issues should be fixed, nice catch, thanks @sixhours! :)

Screenshot 2020-01-28 at 22 49 24

File: client/my-sites/customer-home/main.jsx (hunk 2 above)
Comment (gwwar) [reply client/my-sites/customer-home/main.jsx]: Great catch @sixhours !

**Pull request merged**